### PR TITLE
Export all pgbench pods run outputs to google spreadsheet

### PR DESCRIPTION
**Pull request by PrasadDesala**
Signed-off-by: Prasad Desala <tdesala@redhat.com>

### CODE REVIEW

**Comment by PrasadDesala** [comment]
Attached output screenshot of collected data in spreadsheet:
pgbench_spreadsheet

File: ocs_ci/ocs/pgsql.py [hunk 1]
```diff
@@ -338,6 +340,44 @@ def respin_pgsql_app_pod(self):
...
+        Collect pgbench output to google spreadsheet
+
+        Args:
+            pg_output (list):  pgbench outputs in list
+            sheet_name (str): Name of the sheet
+            sheet_index (int): Index of sheet
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
```
Comment (hnallurv) [inline ocs_ci/ocs/pgsql.py:354]: How are we going to store the results of multiple runs over multiple builds/releases? Is it:
1) one sheet per release with many tabs - each tab for a particular run
 - looks compact
2) one sheet for all the releases with many tabs - each tab for a particular run
 - will have have too many tabs
3) one sheet for each run?
 - will create too many individual files - difficult to browse through and compare
4) any other option?

File: ocs_ci/ocs/pgsql.py [hunk 2]
```diff
@@ -338,6 +340,44 @@ def respin_pgsql_app_pod(self):
...
+            sheet_name='E2E Workloads - PGSQL', sheet_index=sheet_index
+        )
+        log.info("Exporting pgoutput data to google spreadsheet")
+        for i in range(len(pg_output)):
```
Comment (hnallurv) [inline ocs_ci/ocs/pgsql.py:358]: I suggest following:
1) All details like versions of OCP, OCS, ceph etc., to be shown in table format
2) have pgbench pod name in first column and rest of the columns like lat_avg etc., from 2nd column

File: ocs_ci/ocs/pgsql.py (hunk 1 above)
Comment (PrasadDesala) [reply ocs_ci/ocs/pgsql.py:354]: In the spreadsheet "E2E Workloads - PGSQL", I am maintaining individual tabs for each pgsql test. For every run the results of each test gets appended to its own tab. 

Below is the snap of pg_out data of test_pgsql.py for two runs,
multiple_runs

We can collect this way for every run in the interim builds and once we have the final benchmark results we can move the final run details to a new tab and lock(this has to be done manually).

Comment (PrasadDesala) [reply ocs_ci/ocs/pgsql.py:354]: I have updated the patch to only export test_pgsql.

File: ocs_ci/ocs/pgsql.py (hunk 2 above)
Comment (PrasadDesala) [reply ocs_ci/ocs/pgsql.py:358]: Resolved in the new commit

**Comment by PrasadDesala** [comment]
pgbench_data

File: ocs_ci/ocs/pgsql.py [hunk 3]
```diff
@@ -366,21 +378,66 @@ def get_pgbech_pod_status_table(self, pgbench_pods):
...
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
+            sheet_name=sheet_name, sheet_index=sheet_index
+        )
+        log.info("Exporting pgoutput data to google spreadsheet")
+        for i in range(len(pg_output)):
+            for j in range(len(pg_output[i][0])):
+                run_id = list(pg_output[i][0][j].keys())[0]
```
Comment (OdedViner) [inline ocs_ci/ocs/pgsql.py:417]: Can use different variable names? not (i,j)

Comment (PrasadDesala) [reply ocs_ci/ocs/pgsql.py]: Done

**Comment by PrasadDesala** [comment]
new

**Pull request merged**